frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+
+import type { ComponentStory } from "@storybook/react";
+import { useArgs } from "@storybook/client-api";
+
+import styled from "@emotion/styled";
+import stateFixture from "__support__/sample_database_fixture.json";
+import { getMetadata } from "metabase/selectors/metadata";
+import { State } from "metabase-types/store";
+import StructuredQuery from "metabase-lib/queries/StructuredQuery";
+import ExpressionWidget from "./ExpressionWidget";
+
+const state = stateFixture as unknown as State;
```

Comment:
We're planning to eventually get rid of `sample_database_fixture`. Can we compose a meaningful dataset using helpers from `metabase-types/api/mock`? 

<details>
<summary>Example usage</summary>

```js
import {
  createMockDatabase,
  createMockField,
  createMockTable,
} from "metabase-types/api/mocks";
import { TYPE } from "metabase-lib/types/constants";

const field1 = createMockField({ id: 1, name: "ID", semantic_type: TYPE.PK });
const field2 = createMockField({ id: 2, name: "Name" }); 

const table1 = createMockTable({ id: 1, name: "Table", fields: [ field1, field2 ] });

const database = createMockDatabase({ id: 1, tables: [ table1 ] });
```

</details>